src/desktop/components/search_bar/view.coffee

```diff
@@ -147,7 +144,7 @@ module.exports = class SearchBarView extends Backbone.View
   # (rather than actually moving the cursor down
   # which would overwrite the user's typing)
   maybeHighlight: ->
-    @$('.tt-suggestion:first').addClass('tt-cursor') if @autoselect
+    @$('.tt-suggestion:first').addClass('tt-cursor') if @autoselect or (@enableSpotlightAutocomplete and @spotlightSearchResult)
```

Comment:
This is the essential 'fix' - we want to 'select' the first result if we are in autoselect mode (which we are by default- always select the first element), or in spotlight mode, when there is a spotlight result filled in.

Reply:
makes sense :+1: